example-conf/mesos-dns/mesos-site.xml

```diff
@@ -43,7 +43,7 @@
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
-    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
```

Comment:
Yes, that's fine but we need at least 3 Zookeeper nodes for a production quorum. 1 Zookeeper as a default is not ideal.